src/intervals.jl

```diff
@@ -0,0 +1,159 @@
+using .IntervalArithmetic
+
+function evaluate(a::Taylor1, dx::Interval)
+    dx == (-1..1) && return _evaluate(a, dx, Val(true))
```

Comment:
It seems like it is inefficient to be checking this equality each time.

Reply:
I fully agree. Do you have any suggestion which avoids calling directly `_evaluate(a, dx, Val(true))` or `_evaluate(a, dx, Val(false))`? I think this was also mentioned by @mforets, but I haven't come with any better solution...

Reply:
For the time being I'll leave it like this. In another PR we could address this point.